.github/dependabot.yml

```diff
@@ -42,6 +42,11 @@ updates:
     schedule:
       interval: "weekly"
       day: "sunday"
+    ignore:
+      # Exclude Blockly, for more info: https://github.com/code-dot-org/code-dot-org/pull/61094
+      - dependency-name: "@blockly/*"
+      - dependency-name: "blockly"
+      - dependency-name: "pepjs"
```

Comment:
I'm not sure what pepjs does exactly, but it looks like we only use it with `@code-dot-org/blockly` which is our custom fork that we no longer update. I think it would be safe to either continue to include pepjs, or to exclude CDO Blockly in addition.

Reply:
Found some extra context that confirms it was just for CDO Blockly: https://github.com/code-dot-org/code-dot-org/commit/543b1406c6095e7413f925c5ad5e5390efaf6509

In fact, it looks like it was only used for Sprite Lab, which has been on Google Blockly for most of this year. I wonder if we could safely remove it completely.

Thoughts @molly-moen @ebeastlake?

Reply:
But we can probably keep it excluded here and remove it as a separate task

Reply:
@stephenliang Can we add `"@code-dot-org/blockly"` to the exclude list here?